Neos.Kickstarter/Resources/Private/Generator/Fusion/DefaultPrototype.fusion

```diff
@@ -0,0 +1,107 @@
+# The default object used for rendering in most sites
+#
+prototype({packageKey}:{prototypeName}) < prototype(Neos.Fusion:Join)<![CDATA[ {
+
+    doctype = '<!DOCTYPE html>'
+    doctype.@position = 'start 100'
+
+    # Only the opening html tag for the page. This is done to avoid deep nesting of Fusion paths for integrators.
+    htmlTag = Neos.Fusion:Tag {
+        @position = 'start'
+        tagName = 'html'
+        omitClosingTag = true
+    }
+
+    headerComment = ${Configuration.setting(]]>'{packageKey}.headerComment'<![CDATA[)}
+    headerComment.@position = 'before headTag'
+
+    # The opening head tag for the page. This is done to avoid deep nesting of Fusion paths for integrators.
+    headTag = Neos.Fusion:Tag {
+        @position = 'after htmlTag'
+        tagName = 'head'
+        omitClosingTag = true
+    }
+
+    # The content of the head tag, integrators can add their own head content.
+    head = Neos.Fusion:Join {
+        @position = 'after headTag'
+
+        characterSetMetaTag = Neos.Fusion:Tag {
+            @position = 'start 10'
+            tagName = 'meta'
+            attributes {
+                charset = 'UTF-8'
+            }
+        }
+
+        titleTag = Neos.Fusion:Tag {
+            tagName = 'title'
+            # TODO: This should be definable either in the single page Fusion or the action
+            content = ${page-title}
+        }
+
+        # Link tags for stylesheets in the head should go here
+        stylesheets = Neos.Fusion:Join
+
+        # Script includes in the head should go here
+        javascripts = Neos.Fusion:Join {
+            @position = 'after stylesheets'
+        }
+    }
+
+    closingHeadTag = '</head>'
+    closingHeadTag.@position = 'after head'
+
+    # The opening body tag for the page. This is done to avoid deep nesting of Fusion paths for integrators.
+    bodyTag = Neos.Fusion:Tag {
+        @position = 'after closingHeadTag'
+        tagName = 'body'
+        omitClosingTag = true
+    }
+
+    # Content of the body tag. To be defined by the integrator.
+    body = Neos.Fusion:Fragment {
```

Comment:
I *think* the reason I used Fragment was because I needed a fusion object that would not render a tag and I guess I was thinking of React `<Fragment>` - though not sure why I did use `Join` for the head then. Probably a c&p discrepancy.